ledger/participant-integration-api/src/main/scala/platform/apiserver/GrpcServer.scala

```diff
@@ -53,7 +55,16 @@ private[apiserver] object GrpcServer {
     builder.intercept(new MetricsInterceptor(metrics))
     builder.intercept(new TruncatedStatusInterceptor(MaximumStatusDescriptionLength))
     builder.intercept(new ErrorInterceptor)
-    rateLimitingConfig.foreach(c => builder.intercept(new RateLimitingInterceptor(metrics, c)))
+    rateLimitingConfig.foreach(c =>
+      builder.intercept(
+        new RateLimitingInterceptor(
+          metrics = metrics,
+          config = c,
+          memoryPoolMxBeans = ManagementFactory.getMemoryPoolMXBeans.asScala.toList,
```

Comment:
Shouldn't that be an implementation detail invisible to the users of the rate limiting interceptor? We don't need to know how the sausage is made here.